src/components/MapFacetContents.js

```diff
@@ -200,14 +200,24 @@ class MapFacetContents extends React.Component<MapFacetContentsDefaultProps, Map
 
       const points = this.props.buckets.map((bucket) => {
         const value = bucket.value; // JSON.parse(bucket.value);
+        let longitude = 0;
+        let latitude = 0;
+        if (typeof value === 'string') {
+          const valueArr = value.split(',');
+          longitude = valueArr[0];
```

Comment:
These will the the values as strings, not numbers, won't they? You should be able to call Number.parseFloat() to convert them... since the Marker component expects the coordinates property to be of type `Array<number>`.